types/ramda/index.d.ts

```diff
@@ -1159,6 +1160,7 @@ declare namespace R {
          * returned in subsequent invocations.
          */
         once(fn: (...a: any[]) => any): (...a: any[]) => any;
+        once<T>(fn: (...a: any[]) => T): (...a: any[]) => T;
```

Comment:
This overload does not have associated documentation. Consider copying the documentation from the preceding overload.